schemas/test/en/test_currency.json

```diff
@@ -93,10 +92,93 @@
                                         "maximum": {
                                             "value": 1000000
                                         }
+                                    },
+                                    {
+                                        "id": "answer-gbp-max-range",
+                                        "label": "How much did you spend? (Max range)",
```

Comment:
The error message generated when going above the max value is less descriptive ("Number is too large") than the one for going below the minimum ("Enter an answer more than or equal to X"). 
The latter is generated dynamically but I think a generic one specified in this schema is being used instead for the former.

Reply:
I was thinking about removing the generic error message, or making both messages generic. Any preferences you have or other devs ?

Reply:
Actually i shall make it generic since the previous was generic 👍

Reply:
How much bother is it to make it dynamic? The other test cases show dynamic errors, that feels more appropriate for the tester to know they've hit the ranges without having to go back and forth between the schema and tested survey

Reply:
No bother just removing the error messages at the top of the schema. I shall update it so its dynamic

Reply:
Ah nice one, so do error messages defined in the schema take precedence over dynamic ones created in runner?

Reply:
Yeah it should be, I think